includes/admin/reporting/reports.php

```diff
@@ -2548,377 +2547,454 @@ function display_export_report() {
-
-                            <form id="edd-export-sales-earnings" class="edd-export-form edd-import-export-form" method="post">
-								<div class="edd-from-to-wrapper">
+	?>
+	<div id="edd-dashboard-widgets-wrap">
+		<div class="metabox-holder">
+			<div id="post-body">
+				<div id="post-body-content" class="edd-reports-export edd-admin--has-grid">
+
+				<?php do_action( 'edd_reports_tab_export_content_top' ); ?>
+
+				<div class="postbox edd-export-earnings-report">
+					<h2 class="hndle"><span><?php esc_html_e( 'Export Earnings Report', 'easy-digital-downloads' ); ?></span></h2>
+					<div class="inside">
+						<p><?php esc_html_e( 'Download a CSV giving a detailed look into earnings over time.', 'easy-digital-downloads' ); ?></p>
+						<form id="edd-export-earnings" class="edd-export-form edd-import-export-form" method="post">
+							<div class="edd-to-and-from-container">
+							<label for="start_month" class="screen-reader-text">Start Month</label>
+								<?php echo EDD()->html->month_dropdown( 'start_month' ); ?>
+							<label for="start_year" class="screen-reader-text">Start Year</label>
+								<?php echo EDD()->html->year_dropdown( 'start_year' ); ?>
+							</div>
+
+							<span class="edd-to-and-from--separator"><?php echo _x( '&mdash; to &mdash;', 'Date one to date two', 'easy-digital-downloads' ); ?></span>
+
+							<div class="edd-to-and-from-container">
+							<label for="end_month" class="screen-reader-text">End Month</label>
+								<?php echo EDD()->html->month_dropdown( 'end_month' ); ?>
+							<label for="end_year" class="screen-reader-text">End Year</label>
+								<?php echo EDD()->html->year_dropdown( 'end_year' ); ?>
+							</div>
+							<?php wp_nonce_field( 'edd_ajax_export', 'edd_ajax_export' ); ?>
+							<input type="hidden" name="edd-export-class" value="EDD_Batch_Earnings_Report_Export"/>
+							<button type="submit" class="button button-secondary"><?php esc_html_e( 'Generate CSV', 'easy-digital-downloads' ); ?></button>
+						</form>
+					</div>
+				</div>
+
+				<div class="postbox edd-export-sales-earnings">
+					<h2 class="hndle"><span><?php esc_html_e( 'Export Sales and Earnings', 'easy-digital-downloads' ); ?></span></h2>
+					<div class="inside">
+						<p><?php esc_html_e( 'Download a CSV of all sales or earnings on a day-by-day basis.', 'easy-digital-downloads' ); ?></p>
+						<form id="edd-export-sales-earnings" class="edd-export-form edd-import-export-form" method="post">
+							<div class="edd-from-to-wrapper">
+								<label for="edd-order-export-start" class="screen-reader-text">Export Sales and Earnings Start Date</label>
 									<?php
+									echo EDD()->html->date_field(
+										array(
+											'id'          => 'edd-order-export-start',
+											'class'       => 'edd-export-start',
+											'name'        => 'start',
```

Comment:
It looks like the `name` parameter should be made unique for each of these `EDD()->html->date_field()` uses, as that method calls the `EDD()->html->text()` method, which takes the `name` parameter and generates an `id` for a wrapping `span` (`'<span id="edd-' . edd_sanitize_key( $args['name'] ) . '-wrap">'`). So these maybe need to be changed to something like `export-start`, etc. to avoid the span ID duplication (I do not know why the `span` needs an ID at all, but that's a separate question).

Reply:
Do you mean that each of these `date_field()`s should have a unique `id`?  I tried that and it breaks this rule in the CSS:
```
.edd-from-to-wrapper input[name="start"],
.edd-from-to-wrapper input[name="start-date"],
.edd-from-to-wrapper input[name="filter_from"] {
	border-top-right-radius: 0;
	border-bottom-right-radius: 0;
}
```
Do you think it would be better to tweak the CSS (maybe use a wildcard so it's not too verbose) or keep the duplicate ids created for the span?  Was another accessibility issue flagged with the duplicate ids?

Reply:
On the one hand, it may not be worth fixing, because the export screen will still have so many duplicate IDs (the hidden fields are all `#edd_ajax_export`). But it would be nice to fix what we could.

I wonder if we could replace the input styles for the `edd-from-to-wrapper` with something that won't rely on the name element. This _seems_ to work, but would need to be tested to make sure it doesn't break the filter bar and list tables that use it:

```css
.edd-from-to-wrapper span input {
	margin-left: -1px;
	border-top-left-radius: 0;
	border-bottom-left-radius: 0;
}

.edd-from-to-wrapper span:first-of-type input {
	border-top-right-radius: 0;
	border-bottom-right-radius: 0;
	margin-left: 0;
}
```

Alternatively, an even better option would be to fix the `span` ID in the `HTML` class method to either just be a class, which would make more sense to me in general, or to force it to use the `id` attribute if that's supplied, which it often is not, but I think that would more properly fall into a different issue (#7701 might be appropriate).

Reply:
I added this as a comment in #7701, as it seems to address the use of `html->` methods using the `name` attribute